x-pack/plugins/reporting/server/export_types/png_v2/types.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SerializableState } from 'src/plugins/kibana_utils/common';
+import { LocatorParams } from '../../../common/types';
+import { LayoutParams } from '../../lib/layouts';
+import { BaseParams, BasePayload } from '../../types';
+
+interface BaseParamsPNGV2<P extends SerializableState = SerializableState> {
+  layout: LayoutParams;
+  forceNow?: string;
```

Comment:
if we make this a required field, then we need to handle it internally. other option is not having this option on the report and then its up to apps to handle it (make sure their reporting url forces now). i would prefer the later option to keep this report type simple

Reply:
Yeah I was also thinking that this could be a possibility with the export types. Plugins using reporting could also control this by adding it to the state we pass back to them at reporting render time.